CMakeLists.txt

```diff
@@ -603,7 +603,7 @@ if (MSVC)
 endif ()
 
 set (MAKE_SYSTEM)
-if (CMAKE_BUILD_TOOL MATCHES "make")
+if (CMAKE_MAKE_PROGRAM MATCHES "make")
```

Comment:
This brings the change from PR #97 to HDF5 1.12.  It also needs to go to 1.10 and 1.8.  Anywhere else?

Reply:
Yes, when I push these changes down to 1.10 and 1.8.